### PR TITLE
 Added tutorial pages for tool config and HTML report 

### PR DESCRIPTION
**Pull request by psiinon**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
  - Add API endpoints for getting and setting UI options. [#319](https://github.com/zaproxy/zap-hud/issues/319)
  - Add tasks for Enable, Show and Break tutorial pages
  - Add plain text/regex filtering capability to History section [#286](https://github.com/zaproxy/zap-hud/issues/233)
+ - Add tutorial pages for tool configuration anf the HTML report tool.
```
Comment (thc202) [inline CHANGELOG.md]: There's a typo in `and`.

Comment (psiinon) [reply CHANGELOG.md]: Fixed :)

**Comment by thc202** [comment]
Thanks!

**Comment by kingthorin** [comment]
snyk is still "in progress"...

**Comment by thc202** [comment]
It has conflicts now.

**Comment by psiinon** [comment]
Fixed conflicts.

**Pull request merged**